lib/common.js

```diff
@@ -5,12 +5,62 @@ const expressBrowserify = require('express-browserify')
 const { makeTextureAtlas } = require('./atlas')
 const { prepareBlocksStates } = require('./models')
 
+function loadData (versionType, majorVersion) {
+  const dataSource = require('../node_modules/minecraft-data/minecraft-data/data/dataPaths')
+
+  if (dataSource[versionType]) {
+    if (dataSource[versionType][majorVersion]) {
+      const data = []
+
+      for (const dataFile in dataSource[versionType][majorVersion]) {
+        data.push('./minecraft-data/data/' + dataSource[versionType][majorVersion][dataFile] + '/' + dataFile + '.json')
+      }
+
+      return data
+    }
+  }
+
+  return null
+}
+
+function toMajor (mcVersion, preNetty, typeArg) {
```

Comment:
Isn't this function copy pasted from mcData ? It would be better to expose it and use the one from mcData.

Reply:
Yeah, I just copied it directly. I can expose these as well as data sources.